docs/topics/faq.md

```diff
@@ -126,3 +126,9 @@ android {
 ```
 It is discussed in [this issue](https://github.com/takahirom/roborazzi/issues/272).
 Additionally, it might be worth trying to run your tests with VisualVM to monitor memory usage and identify potential leaks.
+
+### Q: Roborazzi's Gradle task is not displayed.
+
+**A:** It is discussed in [this issue](https://github.com/takahirom/roborazzi/issues/493).  
+To enable the display of Roborazzi tasks, please enable ***Configure all Gradle tasks during Gradle Sync (this can make Gradle Sync slower)*** in the settings.  
+<img src="/docs/images/roborazzi_gradle_task_setting.png" width="800" />
```

Comment:
I'm not sure whether this image tag works with our documentation system. 👀

Reply:
The images appear to be displayed in the README of the corresponding branch of the fork repository.
However, I would like to adopt your approach because, as you say, it could cause confusion!

![image](https://github.com/user-attachments/assets/d6ca3d24-fddd-483d-a0e7-7ff7156967cf)

Reply:
The image for use in README
![image](https://github.com/user-attachments/assets/b9cde28b-ddf5-4a55-a171-ea4685f01416)